fastestimator/backend/update_model.py

```diff
@@ -29,7 +29,7 @@ def update_model(model: Union[tf.keras.Model, torch.nn.Module],
             gradients = tape.gradient(loss, model.trainable_variables)
             model.current_optimizer.apply_gradients(zip(gradients, model.trainable_variables))
     elif isinstance(model, torch.nn.Module):
-        loss.backward()
+        loss.backward(retain_graph=True)
```

Comment:
if this flag needs to be set, how was our code working before?

Reply:
this flag is needed if there are multiple UpdateOp in network ops.   by default, the whole graph is released when we call backward. setting it to True will prevent that from happening. 

there's no performance or memory differences from my benchmark.

Reply:
interesting